akka-stream-tests/src/test/scala/akka/stream/io/FileSinkSpec.scala

```diff
@@ -65,12 +65,12 @@ class FileSinkSpec extends StreamSpec(UnboundedMailboxConfig) {
       }, create = false)
     }
 
-    "by default write into existing file" in assertAllStagesStopped {
+    "write into existing file without wiping existing data" in assertAllStagesStopped {
       targetFile { f ⇒
         def write(lines: List[String]) =
           Source(lines)
             .map(ByteString(_))
-            .runWith(FileIO.toPath(f))
+            .runWith(FileIO.toPath(f, Set(StandardOpenOption.WRITE, StandardOpenOption.CREATE)))
```

Comment:
ah, so it overwrites as many bytes as you pass through the stream, but if the file was larger than that the old content is still there after the written position?